### DISCUSSION TITLE
New release v1.3.1 🚀

### DISCUSSION THREAD

**Discussion by diegoferigo** [robotology/gym-ignition#414] [Releases]
Compatible with [Ignition **Fortress**](https://ignitionrobotics.org/docs/fortress/install) (`Gazebo ≥ 6.0.0`).

### Bug fixing and new features
* Fix checking supported Ignition distributions by @diegoferigo in https://github.com/robotology/gym-ignition/pull/405
* Fix Ignition distribution check in case multiple versions are found by @diegoferigo in https://github.com/robotology/gym-ignition/pull/408
* Support conda packaging by @diegoferigo in https://github.com/robotology/gym-ignition/pull/406
* [`devel`]→[`master`] Preparation of v1.3.1 by @diegoferigo in https://github.com/robotology/gym-ignition/pull/411
### Documentation, CI/CD, tests
* Update Ignition installation link to Fortress in README by @movefasta in https://github.com/robotology/gym-ignition/pull/409
* Enable automatic generation of release changelog by @diegoferigo in https://github.com/robotology/gym-ignition/pull/410

## New Contributors
* @movefasta made their first contribution in https://github.com/robotology/gym-ignition/pull/409

**Full Changelog**: https://github.com/robotology/gym-ignition/compare/v1.3.0...v1.3.1

<hr /><em>This discussion was created from the release <a href='https://github.com/robotology/gym-ignition/releases/tag/v1.3.1'>v1.3.1</a>.</em>

**Comment by vfdev-5** [comment]
Current release is incompatible with ignition gazebo 6.1.0. The latest commit fixes versions issue but stable version nor nightly version are not working with latest ignition release. Please release your latest fix.

**Reply by diegoferigo** [reply]
Yep it was fixed in `master` with #415, but I still have to publish a new release. I have in pipeline few things I'd like to merge before, especially the necessary for #407 that is almost ready in [`feature/conda`](https://github.com/robotology/gym-ignition/tree/feature/conda). In the past few days, though, I had not enough time to finalize it.
